swagger/swagger.json

```diff
@@ -11,6 +11,193 @@
   } ],
   "schemes" : [ "https" ],
   "paths" : {
+    "/v1/directdebit/mandates" : {
+      "get" : {
+        "summary" : "Search mandates",
+        "description" : "Searches for mandates with the parameters provided. The Authorisation token needs to be specified in the 'Authorization' header as 'Authorization: Bearer YOUR_API_KEY_HERE'",
+        "operationId" : "searchMandates",
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "name" : "reference",
+          "in" : "query",
+          "required" : false,
+          "type" : "string"
+        }, {
+          "name" : "state",
+          "in" : "query",
+          "required" : false,
+          "type" : "string",
+          "pattern" : "created|started|pending|submitted|active|inactive|cancelled"
```

Comment:
Pretty cool that it picks up the validation annotation like that (I assume this was just automatically generated from the class).